src/Compiler/RefC/CC.idr

```diff
@@ -68,6 +79,8 @@ compileCObjectFile {asLibrary} sourceFile objectFile =
               "-o", objectFile,
               "-I" ++ refcDir,
               "-I" ++ cDir]
+              ++ (words cppFlags)
+              ++ (words cFlags)
```

Comment:
Using `words` here will cause issues with quoted spaces, such as in filenames. This is one of the few cases where we *don't* want to escape a string for a command. It should be concatenated on as a string, after `escapeCmd` has run.

eg. `escapeCmd (...) ++ " " ++ cppFlags ++ " " ++ cFlags`

Reply:
Ah, that's an alternative approach that I hadn't considered! I'll give it a shot and see if it still has the correct behaviour.

Reply:
From a local test, it looks like that also has the behaviour that we want, so I've made the change and I'm running it through CI now.